--- a/commandline/makemake/Makefile
+++ b/commandline/makemake/Makefile
@@ -18,7 +18,8 @@ SHRLIB      := usb_relay_device.so
 # For Linux: Using old simple version 0.1 of libusb
 HIDDATA  := hiddata_libusb01
 USBFLAGS := $(shell libusb-config --cflags)
-USBLIBS  := $(shell libusb-config --libs)
+#USBLIBS  := $(shell libusb-config --libs)
+USBLIBS= -L$(STAGING_DIR)opt/lib -lusb
 EXE_SUFFIX=
 
 
@@ -37,10 +38,10 @@ PROGRAM=	$(CMD_UTILITY)$(EXE_SUFFIX)
 
 all: $(PROGRAM) $(SHRLIB)
 
-CC=		gcc
+#CC=		gcc
 DEBUGFLAGS=
 CFLAGS=		-O -Wall $(USBFLAGS) $(DEBUGFLAGS) -I$(SRCDIR)
-LIBS=		$(USBLIBS)
+LIBS+=		$(USBLIBS)
 OBJ=		$(addsuffix .o,$(SRCS))
 
 
