--- a/commandline/makemake/Makefile
+++ b/commandline/makemake/Makefile
@@ -18,7 +18,8 @@ SHRLIB      := usb_relay_device.so
 # For Linux: Using old simple version 0.1 of libusb
 HIDDATA  := hiddata_libusb01
 USBFLAGS := $(shell libusb-config --cflags)
-USBLIBS  := $(shell libusb-config --libs)
+#USBLIBS  := $(shell libusb-config --libs)
+USBLIBS= -L$(STAGING_DIR)/opt/lib -lusb
 EXE_SUFFIX=
 
 
@@ -37,13 +38,13 @@ PROGRAM=	$(CMD_UTILITY)$(EXE_SUFFIX)
 
 all: $(PROGRAM) $(SHRLIB)
 
-CC=		gcc
+#CC=		gcc
 DEBUGFLAGS=
-CFLAGS=		-O -Wall $(USBFLAGS) $(DEBUGFLAGS) -I$(SRCDIR)
-LIBS=		$(USBLIBS)
+CFLAGS+=		-O $(USBFLAGS) $(DEBUGFLAGS) -I$(SRCDIR)
+LIBS+=		$(USBLIBS) -Wl,-rpath-link=$(STAGING_DIR)/opt/lib
 OBJ=		$(addsuffix .o,$(SRCS))
-
-
+#somcomm
+#somcomm
 $(PROGRAM): $(OBJ)
 	$(CC) -o $(PROGRAM) $(OBJ) $(LIBS)
 
@@ -55,6 +56,13 @@ clean:
 
 %.c.o:
 	$(CC) $(ARCH_COMPILE) $(CFLAGS) -c $*.c -o $*.o
+	
+install:
+#			install ./usbrelay-cmd /usr/local/bin
+#			install ./usb_relay_device.so /usr/local/bin
+#uninstall:
+#			rm -rf /usr/local/bin/usbrelay-cmd	
+#			rm -rf /usr/local/bin/usb_relay_device.so	
 
 #
 # Shared library
