--- a/configure
+++ b/configure
@@ -23050,8 +23050,7 @@
     CPPFLAGS=$(echo  $CPPFLAGS)
     CFLAGS=$(echo $OS_CFLAGS $LIBUSB_CFLAGS $CFLAGS)
     CXXFLAGS=$(echo  $CXXFLAGS)
-    LDFLAGS=$(echo $OS_LDFLAGS $LIBUSB_LIBS
- $LDFLAGS)
+    LDFLAGS=$(echo $OS_LDFLAGS $LIBUSB_LIBS $LDFLAGS)
 
 --- a/src/hid_parsing.c
 +++ b/src/hid_parsing.c
@@ -125,6 +125,8 @@ hid_return hid_find_object(HIDInterface* const hidif,
  ASSERT(hidif->hid_data);
  
  hid_prepare_parse_path(hidif, path, depth);
  
  TRACE("this is debug message");

  if (FindObject(hidif->hid_parser, hidif->hid_data) == 1) {
    NOTICE("found requested item.");