--- a/hidparser/hidparser.c
+++ b/hidparser/hidparser.c
@@ -24,6 +24,7 @@
 
 #include "config.h"
 #include <string.h>
+#include <stdio.h>
 #include "hidparser.h"
 
 /* to be implemented for DEBUG purpose */
@@ -343,8 +344,15 @@ int FindObject(HIDParser* pParser, HIDData* pData)
 {
   HIDData FoundData;
   ResetParser(pParser);
+  
+  printf("debug\n");
+  
   while(HIDParse(pParser, &FoundData))
   {
+	  printf("FoundData 0x%08x \n", (FoundData.Path.Node.UPage << 16)
+          | FoundData.Path.Node.Usage);
+	  printf("pData 0x%08x \n", (pData->Path.Node.UPage << 16)
+          | pData->Path.Node.Usage);
     if(pData->Path.Size>0 && 
       FoundData.Type==pData->Type &&
       memcmp(FoundData.Path.Node, pData->Path.Node, (pData->Path.Size)*sizeof(HIDNode))==0)
