--- a/libhid-0.2.16/hidparser/hidparser.c
+++ b/libhid-0.2.16/hidparser/hidparser.c
@@ -24,6 +24,7 @@
 
 #include "config.h"
 #include <string.h>
+#include <stdio.h>
 #include "hidparser.h"
 
 /* to be implemented for DEBUG purpose */
@@ -343,6 +344,9 @@ int FindObject(HIDParser* pParser, HIDData* pData)
 {
   HIDData FoundData;
   ResetParser(pParser);
+  
+  printf("debug\n");
+  
   while(HIDParse(pParser, &FoundData))
   {
     if(pData->Path.Size>0 && 
